### DISCUSSION TITLE
Image labes

### DISCUSSION THREAD

**Discussion by Startupa** [aaronwmorris/indi-allsky#929] [Q&A]
I've read through the information about the text file used to display labels. Where do I find this file?

I'd to remove some of the items labelled as well as reduce the font size and make them all the same colour. Also, one of the images in the guide shows an area within the server that shows details of the formatting of the labels. I can't find that anywhere.

Thanks

**Comment by aaronwmorris** [comment]
You should be able to adjust the image labels on the config page under the Image tab (close to the bottom).

**Reply by Startupa** [reply]
Thanks, don't know how I missed it!

**Reply by aaronwmorris** [reply]
Hah, I know how you missed it.  That form is huge.  :-)
